test-data/unit/check-multiple-inheritance.test

```diff
@@ -228,7 +228,7 @@ class B(A, int): pass
 from typing import Callable, TypeVar
 T = TypeVar('T')
 class A(B, C):
-    def f(self): pass
+    pass
```

Comment:
Just wanted to note that this change was necessary so that the original error would still be triggered (since redefining the method `f` would mean that the compatibility checks which may trigger the error would be skipped, due to my other changes)